Nukeeper.BitBucketLocal/NuKeeper.BitBucketLocal.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;CA1051;CA1707;CA2227;CA1056;</NoWarn>
```

Comment:
I'd skip `<NoWarn/>`, especially if it's then going to blow up in a Release build.

Reply:
the NoWarn section is a copy from the BitBucket project